FlightAxis/flightaxis.py

```diff
@@ -408,22 +647,52 @@ def test_parse2():
+def signal_handler(sig, frame):
+    print('You pressed Ctrl+C!')
+    print("RC terminating")
+    pygame.quit()
+    fa.saveStates()
+    sys.exit(0)
+
+signal.signal(signal.SIGINT, signal_handler)
+print('Press Ctrl+C')
+
+
+pygame.init()
+pygame.joystick.init()
+
+joystick = pygame.joystick.Joystick(0)
+joystick.init()
+print(f"{joystick.get_name()}")
+
```

Comment:
This is interesting; I learned about `signal` :) This can also be approached as
```python
try:
    # do things
except KeyboardInterrupt:
    # maybe do nothing here
finally:
    # quit stuff
```

Another way, and possibly most preferred, would be to use a context manager
```python
with FlightAxisManager() as axis_manager:
    # do stuff 
```
You can add the setup and cleanup by implementing the `__enter__` and `__exit__` dunder methods.